packages/react/src/components/dropdown-list/dropdown-list.tsx

```diff
@@ -494,6 +505,9 @@ export const DropdownList: VoidFunctionComponent<DropdownListProps<boolean | und
                 value={getValues()}
                 {...dataAttributes /* eslint-disable-line react/jsx-props-no-spreading */}
             >
+                {iconName && (
+                    <TextIcon name={iconName} size={isMobile ? '24' : '16'} data-testid="textboxIcon" />
```

Comment:
Il n'y a pas de regle ecrite, mais le kebab-case gagne en popularite pour les testid. De plus que TagWrapper plus bas utilise le kebab-case.

```suggestion
                    <TextIcon name={iconName} size={isMobile ? '24' : '16'} data-testid="textbox-icon" />
```